README.md

```diff
@@ -33,6 +33,12 @@ Everything is done via a simple GitHub webhook:
 | Content type | `application/json`                                                     |
 | Events       | Send me **everything**.                                                |
 
+#### Additional arguments
+
+| Argument        | Default value      | Description                                     |
+|-----------------|--------------------|-------------------------------------------------|
+| `mute_learners` | `false`            | Prevent _learners_ assignation for this project |
```

Comment:
`mute_learners` _prevents_ assignation.

Shouldn’t the _api_ be a little more explicit? Something like :

```suggestion
| `learners`      | `true`             | Include _learners_ assignation for this project |
```

Reply:
We want `learners` to be assigned by default. It is not an opt-in functionality.

Reply:
I think your right 🤔 

I would change the parameter name to something more explicit like `disable_learners` since `mute` and _Prevent_ does not sound right to me.